index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script>
+      let question;
+
+      function tasuku(work) {
```

Comment:
この名前だと名詞っぽすぎてしまい「何をしてくれる関数なのか」が一見してわかりにくいと思います。
基本的には「自動詞」もしくは「他動詞 + 目的語」にするようにしてみてください。

あと仮引数の `work` もちょっとわかりにくさがある気がします。
タスクには「仕事でないもの」もあるかもしれないので `work` では違和感があります。

```suggestion
      function displayTasks(tasks) {
```